### ISSUE TITLE
Add Nginx_Up metric

### ISSUE THREAD

**Issue by machicolover** [hnlq715/nginx-vts-exporter#82]
Is this possible in this instance of the nginx exporter? This is quite a good metric to be able to scrape in case something happens

**Comment by hnlq715** [comment]
In fact, uptime exists in metrics already.

See the doc [here](https://github.com/hnlq715/nginx-vts-exporter#server-main)

**Comment by machicolover** [comment]
> In fact, uptime exists in metrics already.
> 
> See the doc [here](https://github.com/hnlq715/nginx-vts-exporter#server-main)

Yeah but we will need to be able to monitor if the nginx service went down, not just a dip in uptime. If my first metric can't be done, what should be an alert condition for this using the uptime metric?
